client/src/router.tsx

```diff
@@ -37,6 +37,27 @@ const MainRow = styled.div`
   height: 100%;
 `;
 
+export const isTrackingPromptOpen = (
+  bluetoothName: string | null,
+  shareLocationModalOpen: boolean,
+  publicShareOpen: boolean,
+  centralizeActive: boolean
```

Comment:
Add prefix "is" to all boolean variables and rename centralizeActive to isCentralizationButtonActive(?).